### DISCUSSION TITLE
DynamicDialog v18.0.0-beta.2

### DISCUSSION THREAD

**Discussion by foum-traore** [primefaces/community#2816] [PrimeNG]
Hello,
I'm experiencing an issue with the DynamicDialog: even with modal=true, it's possible to click outside the dialog. In previous versions, the dialog opened with an overlay by default, preventing interactions outside the modal.
Here are the links for comparison: [v18 DynamicDialog Example](https://v18.primeng.org/dynamicdialog#example)
[Previous PrimeNG DynamicDialog Example](https://primeng.org/dynamicdialog#example)

**Comment by a688** [comment]
I have the same issue. p-dialog works correctly but DynamicDialog does not..
